### ISSUE TITLE
NOT FOUND node_modules/vue-context/dist/js/vue-context.js

### ISSUE THREAD

**Issue by danielFlemstrom** [rawilk/vue-context#51]
dist/**js** directory seems to be missing from 5.0.0

When I upgraded from a very early version to 5.0.0 I got the message 
<code>ERROR in ./node_modules/vue-context/dist/js/vue-context.js
Module build failed: Error: ENOENT: no such file or directory, open '/Users/......../node_modules/vue-context/dist/js/vue-context.js'</code>

There is a src/js directory and when I copy that one into dist, the error message disappears but the menu appears before clicking on it 
I also get the following warning:
<code>[Vue warn]: Unknown custom element: <vue-context> - did you register the component correctly? For recursive components, make sure to provide the "name" option.</code>

And obviously, when I right click :
 <code>[Vue warn]: Error in v-on handler: "TypeError: this.$refs.menu.open is not a function"</code>

So it seems the vue-context is not properly recognized. 

4.1.1 though has the proper structure

**Comment by rawilk** [comment]
There is no `dist/js` directory in this version because it's not needed. Also, I'm not sure why you would import from that directory in any previous version as you were always able to import it as `import VueContext from 'vue-context';`, or `import { VueContext } from 'vue-context';` depending on the version.

If you read the documentation, it will detail what you need to know about it. https://vue-context.com/docs/5.0/installation

**Comment by danielFlemstrom** [comment]
Thanks for (amazingly) quick response! I studied the documentation very thoroughly and I believe that I followed the instructions. I never specify the dist/js, so maybe the root cause was some cached stuff. I had to close down the npm run dev instance and when I started again, the error was gone!.

I reverted the previous version and will test the latest when I get time.

Thanks anyway for an awesome component. Very easy to use (when the build framework is not fighting back 😃 )

/Daniel Flemström
